### PR TITLE
Public API documentation integration and Customization

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: app/templates/authenticated/dashboard/public-api.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{page-title 'Public Api'}}
```
Comment (future-pirate-king) [inline app/templates/authenticated/dashboard/public-api.hbs:1]: use translation here

File: app/templates/authenticated/dashboard/public-api/docs.hbs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{{page-title 'Docs'}}
```
Comment (future-pirate-king) [inline app/templates/authenticated/dashboard/public-api/docs.hbs:1]: use translation here

File: app/styles/_public-api.scss
Comment (future-pirate-king) [inline app/styles/_public-api.scss:1]: can we move this to local styles

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/styles/_public-api.scss
Comment (Yibaebi) [reply app/styles/_public-api.scss:1]: Is there any difference or extra advantage of this? 

I think third party style overrides should exist in a single file. It's also easier to maintain, update, and possibly remove styles. I was trying to stick with the same pattern for the freshdesk style overrides.

Comment (future-pirate-king) [reply app/styles/_public-api.scss:1]: advantage is we can target the styles locally then there is no chance of interference with other parts of UI
you can still keep the styles in a single file and share the file reference or through import

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**